src/main/java/org/folio/orders/utils/RequestContextUtil.java

```diff
@@ -18,7 +18,7 @@ public static RequestContext createContextWithNewTenantId(RequestContext request
     }
     var modifiedHeaders = new CaseInsensitiveMap<>(requestContext.getHeaders());
     modifiedHeaders.put(XOkapiHeaders.TENANT, tenantId);
-    logger.info("Request context has been changed with new tenant: {}", tenantId);
+    logger.debug("Request context has been changed with new tenant: {}", tenantId);
```

Comment:
as for me better to remain this log as info, because it shows at lest that me make requests in different tenants